otx/cli/tools/train.py

```diff
@@ -71,24 +66,14 @@ def parse_args():
     parser.add_argument("--data", required=False, default="./data.yaml")
     required = not os.path.exists("./data.yaml")
 
-    parser.add_argument(
```

Comment:
I personally vote for preserving old format as well. Datumaro is a good feature, but this change will break what we have now. Documentation, current datasets format for the experiments.
Let's keep it as optional argument?

Reply:
Unfortunately, if we use the Datumaro, there is no way to input the `--ann-files` directly. Maybe we need to implement additional function to keep our current method